src/components/routes/Account/FirebaseAccountEmailHandler.tsx

```diff
@@ -29,42 +37,97 @@ const DefaultContainer = styled.div`
     section {
       margin-top: 18px;
     }
+
+    .input-container {
+      display: flex;
+      flex-direction: column;
+      margin: 16px 0px 28px 0px;
```

Comment:
i think this is the same as `16px 0 28px`

Reply:
👍